engine/test-util/src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wonder if it would be a good idea to name it logback.xml,
so that we can override this file in some modules without warnings (e.g. when we test deployments we WILL have more INFO logs and we may not want them in this module?

Reply:
It will work with logback-test.xml. Configuration is taken based on order in classpath and order in classpath (I assume) is linearized based on sbt dependencies. I've tested it and it works.